types/react-native/index.d.ts

```diff
@@ -17,6 +17,7 @@
 //                 Ryan Nickel <https://github.com/mrnickel>
 //                 Souvik Ghosh <https://github.com/souvik-ghosh>
 //                 Cheng Gibson <https://github.com/nossbigg>
+//                 Saransh Kataria <https://github.com/saranshkataria>
```

Comment:
Is it clear to you that this means to sign-up as a maintainer and you'll be pinged on each PR/issue for react-native?

Reply:
Yes, I know. Will try and help as much as I can :smile: